targets/gen_noro_nowcasts_targets.R

```diff
@@ -50,21 +51,62 @@ gen_noro_nowcasts_targets <- list(
   ),
   # Forecast quantiles as a scoringutils forecast object
   tar_target(
-    name = quantiled_nowcast_noro,
+    name = su_quantile_noro,
     command = scoringutils::as_forecast_quantile(
-      data = su_forecast_sample,
-      probs = config$Hub_quantiles
+      data = su_sample_noro,
+      probs = config$norovirus$quantiles
     )
   ),
   # Get a wide dataframe with only 50th and 95th for plotting
   tar_target(
     name = summary_nowcast_noro,
-    command = quantiled_nowcast_noro |>
+    command = su_quantile_noro |>
       filter(quantile_level %in% c(0.025, 0.25, 0.5, 0.75, 0.975)) |>
       pivot_wider(
         names_from = "quantile_level",
         values_from = "predicted",
         names_prefix = "q_"
       ) |> left_join(data_as_of, by = "reference_date")
+  ),
+  # Scores--------------------------------------------------------------------
+  tar_target(
+    name = scores_sample_noro,
+    command = scoringutils::score(su_sample_noro)
+  ),
+  tar_target(
+    name = scores_quantile_noro,
+    command = scoringutils::score(su_quantile_noro)
+  ),
+  # Breakdown by overprediction, underprediction, dispersion
+  tar_target(
+    name = observed_data,
+    command = eval_data |>
+      filter(
+        reference_date >=
+          ymd(nowcast_dates_noro) - days(config$norovirus$days_to_eval - 1),
+        reference_date <= ymd(nowcast_dates_noro)
+      ) |>
+      pull(observed),
+    format = "rds"
+  ),
+  tar_target(
+    name = predicted_matrix,
+    command = su_quantile_noro |>
+      select(quantile_level, predicted, reference_date) |>
+      pivot_wider(
+        names_from = quantile_level,
+        values_from = predicted
+      ) |>
+      select(-reference_date) |>
+      as.matrix(),
+    format = "rds"
+  ),
+  tar_target(
```

Comment:
you should be able to get this from modifying the default metrics list vs doing it this way